sw/device/tests/rstmgr_cpu_info_test.c

```diff
@@ -24,179 +23,186 @@ OTTF_DEFINE_TEST_CONFIG();
  * RSTMGR CPU INFO TEST
  *  This test creates a double fault by accessing a register with
  *  a non-existing address.
- *  After the double fault, the dut gets reset by the watch dog bite,
+ *  After the double fault, the DUT gets reset by the watchdog bite
  *  and the test collects / checks the cpu_info from the rstmgr.
  */
 
-// non existing address
-#define kIllegalAddr1 0x4041FFF0u
-#define kIllegalAddr2 0x40003618u
-#define kSkipComp 0x12345678u
 #define kCpuDumpSize 8
```

Comment:
Nit. defines should be named with Captals, but in this case I would move this const to the enum below.

Reply:
Done